domain/src/main/java/org/oppia/android/domain/question/QuestionConstantsProvider.kt

```diff
@@ -21,4 +30,16 @@ class QuestionModule {
   @Provides
   @QuestionTrainingSeed
   fun provideQuestionTrainingSeed(oppiaClock: OppiaClock): Long = oppiaClock.getCurrentTimeMs()
+
+  @Provides
+  @ViewHintPenalty
+  fun provideViewHintPenalty(): Double = 0.1
```

Comment:
Out of curiosity, do these need to be represented as doubles? We could alternatively employ a point system wherein max score is 10 points and penalties are 1 point each. That might let us keep integers everywhere along the computation path which would offer perfect precision, too, until the final division. Would that work?

Reply:
That should work and it makes sense. I believe we went with a decimal system for web and so thats why we translated the same thing here

Reply:
I fixed the implementation to use integers in the score calculation step under the hood (by multiplying everything by a factor of 10), but I'm returning doubles as the result (by dividing everything by 10 again) because Chantel said we may be displaying the scores in fraction form and I wanted to make sure the scores match the decimal system used for web in that case. Let me know what you think!